source/crud/tests/unified/bulkWrite-deleteOne-let.yml

```
@@ -0,0 +1,93 @@
+description: "BulkWrite deleteOne-let"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name crud-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData: &initialData
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents:
+      - { _id: 1 }
+      - { _id: 2 }
+
+tests:
+  - description: "BulkWrite deleteOne with let option"
+    runOnRequirements:
+      - minServerVersion: "5.0"
+    operations:
+      - object: *collection0
+        name: bulkWrite
+        arguments:
+          requests:
+            - deleteOne:
+                filter: &filter
+                  $expr:
+                    $eq: [ "$_id", "$$id" ]
+          let: &let0
+            id: 1
+    expectEvents:
+      - client: *client0
+        events:
+          - commandStartedEvent:
+              command:
+                delete: *collection0Name
+                deletes:
+                  - q: *filter
+                    limit: 1
+                let: *let0
+    outcome:
+      -
+        collectionName: *collection0Name
+        databaseName: *database0Name
+        documents:
+          - { _id: 2 }
+
+  - description: "Bulk Write deleteOne with let option unsupported
+  (server-side error)"
+    runOnRequirements:
+      - minServerVersion: "3.6.0"
+        maxServerVersion: "4.9"
+    operations:
+      - object: *collection0
+        name: bulkWrite
+        arguments:
+          requests:
+            - deleteOne:
+                filter: &filter1
+                  $expr:
+                    $eq: [ "$_id", "$$id" ]
+          let: &let1
+            id: 1
+        expectError:
+          errorContains: "'delete.let' is an unknown field"
+          isClientError: false
+    expectEvents:
+      - client: *client0
+        events:
+          - commandStartedEvent:
+              command:
+                delete: *collection0Name
+                deletes:
+                  -
+                    q: *filter1
```

The line break here doesn't do much to increase readability IMO. Feel free to make similar changes elsewhere. This is similar to what I suggested earlier for `outcome`, but I'm not going to comment on every instance of this.

```suggestion
                  - q: *filter1
```